src/odb/src/codeGenerator/helper.py

```diff
@@ -168,3 +168,8 @@ def get_ref_type(type_name):
         return None
 
     return type_name[6:-1] + "*"
+
+def has_destructible_field(klass):
+  for field in klass["fields"]:
+    return True
+  return False
```

Comment:
What is this for?

Reply:
Was originally for checking the fields but I ended implementing it directly in gen.py. Removed!